go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/coreos/coreos-assembler
 
 go 1.15
+
+require github.com/spf13/cobra v1.5.0
```

Comment:
I'd been thinking we try to avoid a *third* vendor directory in this project; i.e. we change either gangplank/ or mantle/ (or obviously, both) to be Go libraries that are consumed by `/usr/bin/coreos-assembler`.

But...eh.  I wouldn't block this on that either.

Reply:
Yeah. Probably not a problem i'm going to solve here. We wanted golang. We got golang.

Reply:
I think it's not really golang's fault we (before this PR) have *two* distinct vendor/ directories though.

(Anyways I think we're all agreed we can fix this and we're on the right track, so we can close this subthread)